src/Adapter/MSTest.TestAdapter/ObjectModel/FixtureTestResult.cs

```diff
@@ -21,10 +22,5 @@ internal FixtureTestResult(bool isExecuted, UnitTestOutcome outcome, string? exc
     /// <summary>
     /// Gets the outcome of the test.
     /// </summary>
-    public UnitTestOutcome Outcome { get; }
-
-    /// <summary>
-    /// Gets the exception message if any.
-    /// </summary>
-    public string? ExceptionMessage { get; }
```

Comment:
This was never used. Thankfully, `FixtureTestResult` isn't public so we can remove.